test/Kestrel.Transport.Sockets.FunctionalTests/Kestrel.Transport.Sockets.FunctionalTests.csproj

```diff
@@ -7,6 +7,7 @@
     <DefineConstants Condition="$([MSBuild]::IsOSPlatform('OSX'))">$(DefineConstants);MACOS</DefineConstants>
     <ServerGarbageCollection>true</ServerGarbageCollection>
     <TestGroupName>Sockets.FunctionalTests</TestGroupName>
+    <NoWarn>CS8002</NoWarn>
```

Comment:
Unfortunately Selenium.WebDriver isn't strong named so I have to ignore this csc error. Feels strange that we don't do this anywhere else though.